### PR TITLE
added-login and sign-up

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Hello everyone done with the Log-in and Sign-up for my ecommerce.
Please review it. Thanks a lot : )
Live UI(log-in)-https://deploy-preview-5--piston-parts.netlify.app/pages/log-in.html
Live UI(sign-in)-https://deploy-preview-5--piston-parts.netlify.app/pages/sign-in.html

### CODE REVIEW

File: pages/log-in.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                    <button class="btn-icon-sml nav-bottom-icons">
+                        <i class="fas fa-heart"></i></button>
+                </li>
+
+
+            </ul>
+        </div>
+
+
+    </nav>
```
Comment (samyukthasudhakar) [inline pages/log-in.html:68]: Inconsistent line spacing.

File: pages/log-in.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        </div>
+        <div class="nav-top">
+            <ul class="nav-top__list">
+                <li class="nav-top__list-item"> <a href="./" class="link-btn">Sign-in/join</a></li>
+                <li class="nav-top__list-item"><a href="./" class="link-btn">Customer-Care</a></li>
+            </ul>
+        </div>
+        <div class="nav-bottom ">
+            <ul class="nav-bottom__list">
+                <li class="nav-bottom__list-item"><button class="btn nav-bottom-btn">Turbo-chargers</button></li>
```
Comment (samyukthasudhakar) [inline pages/log-in.html:38]: The contents of the nav bar seem to be sticking to the bottom border. Try adding some padding.

**Review by samyukthasudhakar — CHANGES_REQUESTED** [review]
The sign up page's "create new account" link can be changed to "Already have an account, log in >" and be linked to login page. Similarly login page's "create new account" can be linked to sign-up page for better navigation.

File: pages/log-in.html (hunk 1 above)
Comment (Dev-AkshatGupta) [reply pages/log-in.html:68]: > The sign up page's "create new account" link can be changed to "Already have an account, log in >" and be linked to login page. Similarly login page's "create new account" can be linked to sign-up page for better navigation.

Ok thanks for the suggestion I will make the following changes as suggested by you 💯

**Review by foolhardy21 — APPROVED** [review]
LGTM.

**Review by ApurvaSawant11 — CHANGES_REQUESTED** [review]
Log-in and Sign-in are almost the same things. It should be `sign-up` or `register` for creating a new account.

File: pages/sign-in.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            <input type="email" name="email-id" class="form-input" placeholder="Enter Your Email">
+            <input type="password" name="password" class="form-input" placeholder="Enter Your Password">
+
+
+
+            <div><input type="checkbox" name="" id=""><label for="input" class="sub-text">I accept all the terms and
+                    conditions</label></div>
+
+
+            <button class="btn btn-outline-pri form-btn smooth-square-radius ">Sign-In</button>
```
Comment (ApurvaSawant11) [inline pages/sign-in.html:87]: Here it should be `sign up`.

File: pages/log-in.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+            <input type="email" name="email-id" class="form-input" placeholder="Enter Your Email">
+            <input type="password" name="password" class="form-input" placeholder="Enter Your Password">
+
+
+            <div class="flex-center-space-betw">
+                <input type="checkbox" name="" id=""><label for="input" class="sub-text">remember me</label>
+                <a href="" class="link-btn text-accent">Forget password ?</a>
+            </div>
+            <button class="btn btn-outline-pri form-btn smooth-square-radius ">Sign-In</button>
+            <a href="" class="link-btn">Create new account <i class="fas fa-chevron-right text-accent"></i></a>
```
Comment (ApurvaSawant11) [inline pages/log-in.html:87]: If you are using "Login" as your heading then use "Login" for the button as well

Comment (Dev-AkshatGupta) [reply pages/log-in.html:87]: Yeah editing them as you said thanks for the suggestions Apurva 
Thanks for reviewing it too. 💯

**Pull request merged**